shared/AppInsightsCore/src/JavaScriptSDK/BaseCore.ts

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { IAppInsightsCore } from "../JavaScriptSDK.Interfaces/IAppInsightsCore"
+import { IConfiguration } from "../JavaScriptSDK.Interfaces/IConfiguration";
+import { IPlugin, ITelemetryPlugin } from "../JavaScriptSDK.Interfaces/ITelemetryPlugin";
+import { IChannelControls } from "../JavaScriptSDK.Interfaces/IChannelControls";
+import { ITelemetryItem } from "../JavaScriptSDK.Interfaces/ITelemetryItem";
+import { CoreUtils } from "./CoreUtils";
+import { INotificationManager } from '../JavaScriptSDK.Interfaces/INotificationManager';
+import { IDiagnosticLogger } from "../JavaScriptSDK.Interfaces/IDiagnosticLogger";
+import { ChannelController } from './ChannelController';
+
+"use strict";
+
+const validationError = "Extensions must provide callback to initialize";
+
+export class BaseCore implements IAppInsightsCore {
```

Comment:
Add this class export in applicationinsights-core-js.ts so it can be used in other packages

Reply:
yeah this was added already 😁